source/kraken/apply_disruption.cpp

```diff
@@ -182,6 +182,14 @@ struct add_impacts_visitor : public apply_impacts_visitor {
                 r,
                 std::move(stoptimes),
                 pt_data);
+            if (!impact->company_id.empty()) {
+                auto company = pt_data.companies_map[impact->company_id];
```

Comment:
please use 'find(impact->company_id)' instead of '[impact->company_id]'

Reply:
why ? the `[]` operator exist in std::unordered_map.
If you prefer...